doc/OnlineDocs/contributed_packages/sensitivity_toolbox.rst

```diff
@@ -0,0 +1,187 @@
+       
+    Number of Iterations....: 7
+    
+                                       (scaled)                 (unscaled)
+    Objective...............:   4.9999999772520909e-01    4.9999999772520909e-01
+    Dual infeasibility......:   5.5407590737743817e-09    5.5407590737743817e-09
+    Constraint violation....:   8.8817841970012523e-16    8.8817841970012523e-16
+    Complementarity.........:   7.7252085982569584e-09    7.7252085982569584e-09
+    Overall NLP error.......:   7.7252085982569584e-09    7.7252085982569584e-09
+    
+    
+    Number of objective function evaluations             = 8
+    Number of objective gradient evaluations             = 8
+    Number of equality constraint evaluations            = 8
+    Number of inequality constraint evaluations          = 0
+    Number of equality constraint Jacobian evaluations   = 8
+    Number of inequality constraint Jacobian evaluations = 0
+    Number of Lagrangian Hessian evaluations             = 7
+    Total CPU secs in IPOPT (w/o function evaluations)   =      0.002
+    Total CPU secs in NLP function evaluations           =      0.000
+    
+    EXIT: Optimal Solution Found.
+
+We can now inspect the solution :math:`x_1^*`, :math:`x_2^*`, and :math:`x_3^*`:
+
+.. testcode::
+    
+    print("\nOriginal parameter values:")
+    print("    eta1 =",m.eta1())
+    print("    eta2 =",m.eta2())
+    
+    print("Solution with the original parameter values:")
+    print("    x1 =",m.x1())
+    print("    x2 =",m.x2())
+    print("    x3 =",m.x3())
+
+Which gives the output:
+
+.. testoutput::
+    
+    Original parameter values:
+        eta1 = 4.5
+        eta2 = 1.0
+    Solution with the original parameter values:
+        x1 = 0.15
+        x2 = 0.15
+        x3 = 0.0
+
+Likewise, we can inspect the approximate solution :math:`\hat{x}_1^*`, :math:`\hat{x}_2^*`, and :math:`\hat{x}_3^*`:
+
+.. testcode::
+    
+    print("\nNew parameter values:")
+    print("    eta1 =",m_sipopt.perturbed_eta1())
+    print("    eta2 =",m_sipopt.perturbed_eta2())
+    
+    print("(Approximate) solution with the new parameter values:")
+    print("    x1 =",m_sipopt.x1())
+    print("    x2 =",m_sipopt.x2())
+    print("    x3 =",m_sipopt.x3())
+
+Which gives the output:
+
+.. testoutput::
+    
+    New parameter values:
+        eta1 = 4.0
+        eta2 = 1.0
+    (Approximate) solution with the new parameter values:
+        x1 = 0.5000000037913185
+        x2 = 0.4999999939338906
+        x3 = 0.0
+
+Installing sIPOPT
+-----------------
+
+The sensitivity toolbox requires sIPOPT is installed and available in your system PATH. See the IPOPT documentation for detailed instructions:
+
+* https://coin-or.github.io/Ipopt/INSTALL.html
+* https://projects.coin-or.org/Ipopt/wiki/sIpopt
+* https://coin-or.github.io/coinbrew/
+
+Tip: If you get an error that ``ipopt_sens`` cannot be found, you need to make sure sIPOPT was installed and that it is in the system path.
```

Comment:
I think I would change this to a `.. note::`

Reply:
Good suggestion.